dsperse/src/run/utils/runner_utils.py

```
@@ -0,0 +1,341 @@
+            try:
+                Converter.convert(str(slices_root), output_type=original_format, cleanup=True)
+            except Exception as e:
+                logger.warning(f"Failed to repackage to {original_format}: {e}")
+
+    @staticmethod
+    def _get_file_path() -> str:
+        """Get the parent directory path of the current file."""
+        return os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+    @staticmethod
+    def preprocess_input(input_path:str, model_directory: str = None, save_reshape: bool = False) -> torch.Tensor:
```

Ruff (0.14.5)

Unused static method argument: `model_directory`

(ARG004)